src/marqo/tensor_search/tensor_search.py

```diff
@@ -1535,17 +1550,24 @@ def get_query_vectors_from_jobs(queries: List[BulkSearchQueryEntity], qidx_to_jo
             # multiple queries. We have to weight and combine them:
             index_info = get_index_info(config=config, index_name=q.index)
             weighted_vectors = [np.asarray(vec) * weight for vec, weight in zip(vectors, [w for _, w in ordered_queries])]
-            vectorised_text = np.mean(weighted_vectors, axis=0)
+            merged_vector = np.mean(weighted_vectors, axis=0)
             if index_info.index_settings['index_defaults']['normalize_embeddings']:
-                norm = np.linalg.norm(vectorised_text, axis=-1, keepdims=True)
+                norm = np.linalg.norm(merged_vector, axis=-1, keepdims=True)
                 if norm > 0:
-                    vectorised_text /= np.linalg.norm(vectorised_text, axis=-1, keepdims=True)
-            result[qidx] = list(vectorised_text)
+                    merged_vector /= np.linalg.norm(merged_vector, axis=-1, keepdims=True)
+            result[qidx] = list(merged_vector)
         else:
             result[qidx] = vectors[0]
     return result
 
-def _bulk_vector_text_search(config: Config, queries: List[BulkSearchQueryEntity], device=None, raise_on_searchable_attribs=False, result_count: int = 5) -> List[Dict]:
+def create_empty_query_response(queries: List[BulkSearchQueryEntity]) -> List[Dict]:
+    return list(
+        map(
+            lambda x: {"hits": []}, queries
+        )
+    )
+
+def _bulk_vector_text_search(config: Config, queries: List[BulkSearchQueryEntity], device=None) -> List[Dict]:
```

Comment:
what is this function?

Reply:
Will add docstring. This is the equivalent of `_vector_text_search(`